src/ert/_c_wrappers/enkf/lark_parser.py

```diff
@@ -45,6 +46,25 @@ def __str__(self):
         return self.value
 
 
+def assert_is_token(
+    val: Any, kw: Optional[FileContextToken] = None
+) -> TypeGuard[FileContextToken]:
```

Comment:
Please create a test that checks that the typeguards actually create reasonable error messages.